Apps/Common/src/Services/AccessTokenService.cs

```diff
@@ -72,25 +72,26 @@ public class AccessTokenService : IAccessTokenService
         public async Task<RequestResult<TokenSwapResponse>> GetPhsaAccessToken()
         {
             using Activity? activity = Source.StartActivity();
-            string? hdid = this.authenticationDelegate.FetchAuthenticatedUserHdid();
+            string? identifier = this.authenticationDelegate.FetchAuthenticatedUserIdentifier();
             RequestResult<TokenSwapResponse> requestResult = new();
-            TokenSwapResponse? cachedAccessToken = this.GetFromCache(hdid);
+            string cacheKey = $"{TokenSwapCacheDomain}:Identifier:{identifier}";
```

Comment:
I don't think we need to use Identifier in the cacheKey, hdid actually meant something but sub is less useful.   \${TokenSwapCacheDomain}:{userId} is probably better.

Reply:
Changed.